app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -22,20 +19,30 @@ class Api::V2::SubscriptionsController < Api::V2::ApiController
     api :GET, "/organizations/:organization_id/subscriptions", N_("List organization subscriptions")
     api :GET, "/systems/:system_id/subscriptions", N_("List a system's subscriptions"), :deprecated => true
     api :GET, "/activation_keys/:activation_key_id/subscriptions", N_("List an activation key's subscriptions")
+    api :GET, "/subscriptions"
+    param_group :search, Api::V2::ApiController
     param :organization_id, :number, :desc => N_("Organization ID"), :required => true
     param :system_id, String, :desc => N_("UUID of the system"), :required => false
     param :activation_key_id, String, :desc => N_("Activation key ID"), :required => false
-    param_group :search, Api::V2::ApiController
+    param :available_for, String, :desc => N_("Object to show subscriptions available for, either 'system' or 'activation_key'"), :required => false
+    param :match_system, :bool, :desc => N_("Return subscriptions that match system")
+    param :match_installed, :bool, :desc => N_("Return subscriptions that match installed")
+    param :no_overlap, :bool, :desc => N_("Return subscriptions that don't overlap")
```

Comment:
I moved that with the match_installed and match_system as well, but I can change that parameter to overlap.